### ISSUE TITLE
your first model extension

### ISSUE THREAD

**Issue by donotdespair** [russellyang728/mcxs24-russyang#3]
Hey @russellyang728 

I was thinking of your potential model extension. The one we discussed is alright, but we could simplify this thing a bit for you. If you want to hear my proposal drop by the consultations. It will cut down the dimensionality of the hierarchical prior from matrix-variate normal to univariate normal. Interested?

Greetings, T

**Comment by russellyang728** [comment]
Yes @donotdespair , I am definitely interested in this idea. But the conflict is I have another group project assessment which I need finish by tomorrow night. Now it has been halfway done.  I believe I can finish today.
For our submission 2, I started working on the algorithm already. I do have some questions. Do you mind if I go to your office tomorrow afternoon instead today?

**Comment by donotdespair** [comment]
Of, course, @russellyang728 ! See you then!
